gr-vocoder/lib/freedv_tx_ss_impl.cc

```diff
@@ -82,7 +81,7 @@ freedv_tx_ss_impl::freedv_tx_ss_impl(int mode,
     if ((d_freedv = freedv_open(mode)) == NULL)
         throw std::runtime_error("freedv_tx_ss_impl: freedv_open failed");
 #endif
-    snprintf(d_cb_state.tx_str, 79, "%s", d_msg_text.c_str());
+    snprintf(d_cb_state.tx_str, 80, "%s\r", d_msg_text.c_str());
```

Comment:
please investigate max length of freedv payload

Reply:
so this fix actually overflows and doesn't catch the carriage return. I'll find another solution and resubmit.